source/_components/watson_tts.markdown

```diff
@@ -0,0 +1,110 @@
+---
```

Comment:
Pherhaps also add the IOT class

Reply:
Which class should it be? I checked the other TTS services and no other has this field.

Reply:
@klaasnicolaas :point_up: ?

Reply:
If we don't know then we can leave it out.

Reply:
Okay then